### PR TITLE
Removing the automate-cds service from the automate product

### PR DESCRIPTION
**Pull request by lancewf**
### :nut_and_bolt: Description: What code changed, and why?
Removing the automate-cds service from the "automate" product. The automate-cds service as unintentionally added to the "automate" product by updating the products.meta file and not running the "deployment_generate" command. This is also adding a test so that does not happen again. 

### :+1: Definition of Done

The automate-cds service does not start on normally Automate installs. 

### :athletic_shoe: How to Build and Test the Change
1. Remove the `products = ["automate-dev"]` line from the dev/config.toml file. 
1. Run `start_all_services`
1. Ensure the automate-cds is not running. 

### :white_check_mark: Checklist

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [x] Tests added/updated?
- [ ] Docs added/updated?
- [x] All commits have been signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: .studio/common [hunk 1]
```diff
@@ -179,6 +179,24 @@ function verify_all_protobuf_components() {
...
+  git add .
+  git diff --staged --exit-code --ignore-submodules=all
+  result=$?
+
+  if [ $result != 0 ]; then
+    echo "Run 'deployment_generate' to fix this problem"
+    return $result
+  fi
+}
+
```
Comment (lancewf) [inline .studio/common:199]: This test prevents this from happening again.

File: components/automate-deployment/pkg/services/internal/generated/gen.go [hunk 2]
```diff
@@ -541,7 +540,9 @@ var ProductMetadataJSON = `
       "name": "automate-dev",
       "aliases": null,
       "type": "product",
-      "services": [],
+      "services": [
```
Comment (lancewf) [inline components/automate-deployment/pkg/services/internal/generated/gen.go:543]: This file is out of sync with the /products.meta file.

File: integration/tests/security.sh [hunk 3]
```diff
@@ -13,7 +13,7 @@ do_deploy() {
         --manifest-dir "$test_manifest_path" \
         --admin-password chefautomate \
         --product workflow \
-        --product automate \
+        --product automate-dev \
```
Comment (lancewf) [inline integration/tests/security.sh:16]: This test needs all the services running.

File: .expeditor/verify.pipeline.yml [hunk 4]
```diff
@@ -24,6 +24,25 @@ steps:
             - HAB_STUDIO_SUP=false
             - HAB_NONINTERACTIVE=true
 
+  #
+  # A2 products generation - Detect if the products.meta or package.meta files has been up without generating
```
Comment (yzl) [inline .expeditor/verify.pipeline.yml:28]: [nit] has been updated?

**Pull request merged**